spring/boot1-autoconfigure/build.gradle

```diff
@@ -76,3 +82,9 @@ tasks.compileTestJava.dependsOn(project(':spring:boot2-autoconfigure').tasks.com
 tasks.withType(Checkstyle) {
     onlyIf { false }
 }
+
+// Graceful shutdown is supported only on Spring Boot 2
+// because Spring Boot 1 cannot convert a String to a Duration.
+tasks.withType(Test) {
+    exclude '**/ArmeriaGracefulShutdownConfigurationTest*'
+}
```

Comment:
@heowc You might be interested in this :wink:

Reply:
Oh... I'll check 😱

Reply:
I think this is a side effect due to a problem from before it was created.
The current Spring-related dependency is looking at 2.x, not 1.5.x. I don't know when this has been done, but it has been going on before and the tests I've written passed normally. 

This means that with the addition of [these lines](https://github.com/line/armeria/pull/2945/files#diff-53dbe37e6482ebcaaa24ae04f6b9e1abR35-R40), the dependencies have returned to normal and the problem has only come to light. I think I should definitely fix this. WDYT?

Reply:
Yep, that's correct. We can fix this by providing string-to-duration converter for older spring boot.

Reply:
I will make a separate PR and fix it. 😭